index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
+        <nav class="nav-bar">
+            <a href="#" class="logo-name">Seth Bizimana</a>
+           <a href="#" class="menu-icon">
+                <img alt="icon of menu button" src="images/Union.png">
+           </a> 
+        </nav>
+    </header>
+    <main class="main">
+        
+        
+        <article class="headline">
+            <header class="main-header">
+                <span>Hello, This is Seth</span> 
+                <h2>I'am a Software Developer</h2>
+            </header>
+            <p class="headline-desc">
+                Ican help you build a product, feature or website
+                through somem of my work and experience! 
+                if you like what you see and intereseted to work
+                with me, reach me via:
+            </p>
+            <div class="social-icons">
+                <img alt="social media icons" src="images/social_media.png">
+            </div>
+        </article>
+    </main>
+    
+</body>
```

Comment:
- [ ] Make your website look similar/closer to the design. You can see that there are some differences like, the fonts size, spaces, image positioning, etc. 

Your design | Figma design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/170456826-da93713e-23f5-46a4-9e3c-3cf9230c233d.png)  | ![image](https://user-images.githubusercontent.com/43877689/170455915-d05ed4d8-095f-4fd8-a8ae-444f4231d62b.png)

- [ ] Your website is not well-fitting on the mobile screen as well, try and fix it. Maybe I would recommend you to use flexbox very well and let your website spread the whole page so that when you minimize on a mobile device, it will fit very well.